cves/CVE-2013-2135.yml

```diff
@@ -117,20 +132,26 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
-  date:
-  automated:
-  apache:
-  contest:
+  answer: |
+    The vulnerability was discovered and reported by Jon Passki from Coverity Security Research Laboratory.
+    There is no information available on how the vulnerability was discovered.
```

Comment:
I think there was more information available on a blog post, but the forum has since been taken down.

Reply:
If we're thinking of the same blog post, it's still available [here](https://web.archive.org/web/20130823042415/https://communities.coverity.com/blogs/security/2013/05/29/struts2-remote-code-execution-via-ognl-injection) but does not provide any information relating to discovery methodology.